components/sandbox/tiers/staging/konflux-eaas/tiertemplate-konflux-eaas-clusterresources-222461918-222461918.yaml

```diff
@@ -0,0 +1,31 @@
+# ----------------------------------------------------------------
+# Generated by cli - DO NOT EDIT
+# ----------------------------------------------------------------
+
+apiVersion: toolchain.dev.openshift.com/v1alpha1
+kind: TierTemplate
+metadata:
+  name: konflux-eaas-clusterresources-222461918-222461918
+  namespace: toolchain-host-operator
+spec:
+  revision: 222461918-222461918
+  template:
+    apiVersion: template.openshift.io/v1
+    kind: Template
+    metadata:
+      creationTimestamp: null
```

Comment:
This is still not good, you will need to use my changes to ksctl here https://github.com/kubesaw/ksctl/pull/50/files

Reply:
That worked, thanks!